payroll/models/hr_payslip.py

```diff
@@ -483,6 +484,11 @@ def get_current_contract_dict(self, contract, contracts):
 
         return {}
 
+    def _get_tools_dict(self):
+        # _get_tools_dict() is intended to be inherited by other private modules
+        # to add tools or python libraries available in localdict
+        return {"math": math}  # "math" object is useful for doing calculations
```

Comment:
I prefer that OCA/payroll returns an empty dict. There is no compelling case to have `math` in the base module but not some other python module. People who need this functionality can override this method in their own modules. If your really need this to be part of OCA/Payroll I would also be okay with creating another module in the repo to provide this functionality separately.

Reply:
Sorry, I take that back. After thinking about it some more I can see how having some basic methods provided by `math` can be useful without having to install an additional one-line module.

Reply:
Yes i think math is very useful in salary rules. There are other modules that can be useful but i just keep "math" here because don't want to overload this. Also, it serves as an example of how to inherit and use this method. 

I have some rules that i can refactor using math, that's why i come up with the idea of allowing injecting libraries to the localdict. 

Great you liked it!